straxen/scada.py

```diff
@@ -88,11 +91,11 @@ def get_scada_values(self,
         :param filling_kwargs: Kwargs applied to pandas .ffill() or
             .interpolate().
         :param down_sampling: Boolean which indicates whether to
-            donw_sample result or to apply moving average. Moving average
+            donw_sample result or to apply average. The averaging
             is deactivated in case of interpolated data.
-        :param value_every_seconds: Defines with which time difference
-            values should be returned. Must be an integer!
-            Default: one value per 1 seconds.
+        :param every_nth_value: Defines over how many values we compute
+            the average or the nthed sample in case we down sample the
```

Comment:
nth sample?

Reply:
Do you refer to "or the nthed sample in case we down sample" ? 

What I mean you basically do something like `[::nthed_sample]`

Reply:
Doesn't matter much but I would not say `nthed` sample but `nth` sample. Like you say 5th sample and and not 5thed sample.

Reply:
Ah yes sure.